SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -82,5 +82,21 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** Change 'Load' value in inspector to represent worker Frame Time instead of a fraction of target FPS.*/
 	UPROPERTY(EditAnywhere, config, Category = "Metrics", meta = (ConfigRestartRequired = false))
 	bool bUseFrameTimeAsLoad;
+
+	/** If the Development Authentication Flow is used, the client will try to connect to the cloud rather than local deployment. */
+	UPROPERTY(EditAnywhere, config, Category = "Cloud Test", meta = (ConfigRestartRequired = false))
```

Comment:
"Cloud Test" sounds a little temporary. Perhaps "Cloud Development" or "Cloud Connection" ?

Reply:
+1 that